Automattic/newspack-network
# feat(node): added wp cli command - process pending requests

### All Submissions:


### Changes proposed in this Pull Request:

This PR introduces a WP CLI command for processing `np_webhook_request`s with post-meta status in `pending` state. 

### How to test the changes in this Pull Request:

1. Ensure a Hub and Node relationship exists and is configured correctly.
2. Node: Add incorrect "Hub URL" - WP Admin > Newspack Network > Node Settings
3. Node: Perform individual (as many as you can, the more the better) user related updates i.e. update; name, lastname etc.
4. Node: Add correct "Hub URL" - WP Admin > Newspack Network > Node Settings
5. Open terminal and run `wp newspack-network process-webhooks` to process and delete pending requests

### Other information:
